### ISSUE TITLE
background how to open Google Authenticato

### ISSUE THREAD

**Issue by niudin** [matrix-org/synapse#15894]
Please don't file github issues asking for support.
How to enable Google Authenticator in the background, so that individual users need to enter MFA when logging in
Instead, please join [`#synapse:matrix.org`](https://matrix.to/#/#synapse:matrix.org)
(from a matrix.org account if necessary), and ask there.

**Comment by clokep** [comment]
Matrix logins don't currently support MFA, see proposals to add it: [MSC1998](https://github.com/matrix-org/matrix-spec-proposals/pull/1998) or [MSC2771](https://github.com/matrix-org/matrix-spec-proposals/pull/2771), although it is more likely that this will come via implementing OpenID Connect (see [MSC3861](https://github.com/matrix-org/matrix-spec-proposals/pull/3861)).

tl;dr the Matrix spec would need to be updated to support MFA first before Synapse could do anything about it.

**Comment by niudin** [comment]
Please how time support MFA

在 2023年7月10日星期一，Patrick Cloke ***@***.***> 写道：
